.github/workflows/deploy.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [ 3.9, '3.10', '3.11', '3.12']
+        python-version: ['3.10', '3.11', '3.12']
```

Comment:
Will we introduce `3.13` after the `qibo` release? Currently `pip` complains about it when installing `qibo` from source.

Reply:
oh yeah you're right. I forgot about that. Sorry.